karma.conf.js

```diff
@@ -94,6 +109,12 @@ function combinedLoaders() {
     case 'istanbulInstrumenter':
     case 'tslint':
       return aggregate;
+    case 'ts':
+      return aggregate;
+    case 'tsTest':
+      return aggregate.concat([ // force inline source maps
+        Object.assign(loaders[k],
+          { query: { babelOptions: { sourceMaps: 'both' } } })]);
```

Comment:
This option has already been set inside of `.babelrc`. Was it not respecting that option?